ViewModel/EnvironmentAndExtendStoreUuid.php

```diff
@@ -86,6 +86,37 @@ public function getExtendStoreUuid(): ?string
         }
     }
 
+    public function isExtendProductProtectionEnabled(): bool
+    {
+        return $this->scopeConfig->getValue(Extend::ENABLE_PRODUCT_PROTECTION) === '1';
+    }
+
+    public function isProductProtectionProductDisplayPageOfferEnabled(): bool
+    {
+        return $this->scopeConfig->getValue(
+            Extend::ENABLE_PRODUCT_PROTECTION_PRODUCT_DISPLAY_PAGE_OFFER
+        ) === '1';
+    }
+
+    public function isProductProtectionCartOfferEnabled(): bool
+    {
+        return $this->scopeConfig->getValue(Extend::ENABLE_PRODUCT_PROTECTION_CART_OFFER) === '1';
+    }
+
+    public function isProductProtectionPostPurchaseLeadModalOfferEnabled(): bool
+    {
+        return $this->scopeConfig->getValue(
+            Extend::ENABLE_PRODUCT_PROTECTION_POST_PURCHASE_LEAD_MODAL_OFFER
+        ) === '1';
+    }
+
+    public function isProductProtectionProductCatalogPageModalOfferEnabled(): bool
+    {
+        return $this->scopeConfig->getValue(
+            Extend::ENABLE_PRODUCT_PROTECTION_PRODUCT_CATALOG_PAGE_MODAL_OFFER
+        ) === '1';
+    }
+
```

Comment:
need to add these helper functions in order to access the relevant config values within the phtml templates + determine whether offers should be rendered or not. this pattern is lifted from the guidance module.

Reply:
What happens if there is no value, since you typed the return as bool?

Reply:
@scott-taylor-extend if there is no persisted value for this path in `core_config_data` the framework will return the default system value specified in `config.xml` which is `'1'`, so the comparison will run successfully and evaluate to `true`